### DISCUSSION TITLE
[SUGGESTION] Open an existing project

### DISCUSSION THREAD

**Discussion by XenBG** [TiagoSilvaPereira/vemto-issues#135] [Ideas & Suggestions]
It willl be nice if we can open an existing project and the app to show us everything already been created in that project (like Controllers, Models, etc.). In that way we should be able to make changes of variable names, database tables, relations and etc.

I don't know if that's possible because every project have their own file (and maybe logical) structure, so if we just have an option **ONLY** to make changes to our database tables and their relations, I'll be more than happy.

**Comment by TiagoSilvaPereira** [comment]
@XenBG thank you for your suggestion... I wish to add this feature in the future. Have a nice day
